### ISSUE TITLE
Remove acc output for regression problem?

### ISSUE THREAD

**Issue by icyblade** [fchollet/keras#4308]
As @fchollet have pointed out in #724:

> accuracy is not relevant at all for a regression problem

Is it possible to remove accuracy output for regression problems as it might be misleading to newbies?

**Comment by carlthome** [comment]
Hmm? Just don't add `metrics=['accuracy']` when calling `model.fit()`. Doesn't that do it?

**Comment by icyblade** [comment]
The document says:
> Typically you will use metrics=['accuracy']

So I always use .`compile('rmsprop', 'msle', ['accuracy'])` for my network.
Thanks for your help!
